shell/platform/android/io/flutter/embedding/android/FlutterSurfaceView.java

```diff
@@ -56,6 +58,35 @@ public void surfaceCreated(@NonNull SurfaceHolder holder) {
 
           if (shouldNotify()) {
             connectSurfaceToRenderer();
+
+            // Workaround for https://github.com/flutter/flutter/issues/139630
+            // We should not need this if Samsung fixes b/316626640
```

Comment:
"b/316626640" is a Google-internal-ism.

Reply:
Yes - this bug is only visible to Googlers and (some?) Samsung employees unfortunately. I've updated the text around it, but I'd like to keep it here for people who have access.

Reply:
@Hixie @dnfield 

I would like to know what the current progress/status is regarding this Google internal issue.
"b/316626640"

Can you please let me know?